src/axom/spin/BVH.hpp

```diff
@@ -364,24 +332,195 @@ class BVH
 private:
   /// \name Private Members
   /// @{
+  static constexpr FloatType DEFAULT_SCALE_FACTOR = 1.001;
```

Comment:
Minor: I ran into some weird cases in the InOutOctree class that needed a less "regular" default spacing.
The following seemed to work better: `1.000123`

This might not be an issue for the BVH class, but it would be worthwhile testing this against the following mesh: 
https://github.com/LLNL/axom_data/blob/main/quest/cap.stl

See also:
 * https://github.com/LLNL/axom/blob/7e2c27b2229e2778281e755fc0d43bcf93119cdd/src/axom/quest/InOutOctree.hpp#L429
* https://github.com/LLNL/axom/issues/513

Reply:
I agree with Kenny's suggestion.

Reply:
Done 👍